### DISCUSSION TITLE
Player view focused on album art navigation

### DISCUSSION THREAD

**Discussion by s0ur4v116** [fooyin/fooyin#405] [Ideas]
Will it be possible to make navigation centered on album art? For example, something like this:
![image](https://github.com/user-attachments/assets/26793419-089d-4459-9381-a6e2af82d828)

I listen to a lot of japanese music, but unfortunately since I do not know the language, I cannot really know which song is which based on the artist name or album name. I feel like in cases like these, an album art focused navigation system would be much better.

**Comment by Not-Sure-Whats-Going-On** [comment]
Yes please - this, definitely.  It seems the natural way to navigate large collections of albums is by browsing the covers.

**Comment by KRadiants** [comment]
This would be very helpful indeed

**Comment by MarcoCacoete** [comment]
As a musicbee user I tried making one, turned out nice for my purposes.

<img width="1720" height="1398" alt="2026-02-08_03-16" src="https://github.com/user-attachments/assets/d13b0917-bc1e-4aa7-bedd-b115c281c844" />

[Full.fyl.zip](https://github.com/user-attachments/files/25157586/Full.fyl.zip)

**Comment by dcog989** [comment]
This is all that is really missing from Fooyin for me. I want a 'wall of art' of [Album artist thumbnail, either their image or a random album cover] > [album covers] > [track list].

As OP says this is the 'natural' way to view a large collection of albums - but no movement on this since 2024??

I'm new to Fooyin - maybe it's possible now?
